modules/autocomplete-feed/test/spec.coffee

```diff
@@ -94,7 +94,7 @@ describe 'autocomplete-feed', ->
       af.items(items)
       callback = chai.spy()
       af.filter('a', callback)
-      callback.should.have.been.called.once()
+      callback.should.have.been.called.once
```

Comment:
Updating Chai meant changing to use the updated API which does not treat these as functions